.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: ci
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  push:
```

Comment:
If CI doesn't run on `master`, do the commits on `master` get checked? There might be issues because PRs are not always in sync.

Reply:
At that time, I used to think that `master` is checked daily by schedule trigger, so it's not necessary to check when merging PR `master` immediately. But since it's not necessary to remove `master` trigger, I've recovered it.